### PR TITLE
Add namnx228 as reviewer

### PR DESCRIPTION
**Pull request by furkatgofurov7**

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -12,3 +12,4 @@ reviewers:
  - maelk
  - stbenjam
  - zaneb
+ - namnx228
```
Comment (dtantsur) [inline OWNERS:15]: Let's keep the list sorted?

Comment (furkatgofurov7) [reply OWNERS:15]: Sure, you mean alphabetically?

Comment (dtantsur) [reply OWNERS:15]: yep

Comment (furkatgofurov7) [reply OWNERS]: @dtantsur done.

**Review by elfosardo — APPROVED** [review]

**Pull request merged**